docs/zoo/index.md

```diff
@@ -20,33 +20,39 @@ _Larq Zoo is part of a family of libraries for BNN development; you can also che
 
 ## Available models
 
-The following models are trained on the [ImageNet](http://image-net.org/) dataset. The Top-1 and Top-5 accuracy refers to the model's performance on the ImageNet validation dataset, memory refers to the memory after quantization of the weights.
+The following models are trained on the [ImageNet](http://image-net.org/) dataset.
+The Top-1 and Top-5 accuracy refers to the model's performance on the ImageNet validation dataset, memory refers to the memory after quantization of the weights.
+Models were benchmarked using [Larq Compute Engine](/compute-engine/) on a [Pixel 1 phone (2016)](https://support.google.com/pixelphone/answer/7158570?hl=en-GB), single-threaded.
 
 The model definitions and the train loops are available in the [Larq Zoo repository](https://github.com/larq/zoo).
 
 The [`sota`](/zoo/api/sota/) submodule contains these models:
 
-| Model                                         | Top-1 Accuracy | Top-5 Accuracy | Parameters | Memory  |
-| --------------------------------------------- | -------------- | -------------- | ---------- | ------- |
-| [QuickNet](/zoo/api/sota/#quicknet)           | 58.6 %         | 81.0 %         | 10 511 232 | 3.18 MB |
-| [QuickNetLarge](/zoo/api/sota/#quicknetlarge) | 62.7 %         | 84.0 %         | 11 819 136 | 4.49 MB |
-| [QuickNetXL](/zoo/api/sota/#quicknetxl)       | 67.0 %         | 87.3 %         | 22 058 368 | 6.22 MB |
+| Model                                         | Top-1 Accuracy | Top-5 Accuracy  | Model size | Latency (Pixel 1, single thread)[^1] |
```

Comment:
How do you feel about moving the footnote to line 25? I think that might be a bit cleaner.